frame/contracts/src/exec.rs

```diff
@@ -124,7 +130,7 @@ pub trait Ext {
 		code_hash: CodeHash<Self::T>,
 		rent_allowance: BalanceOf<Self::T>,
 		delta: Vec<StorageKey>,
-	) -> DispatchResult;
+	) -> Result<(u32, u32), (DispatchError, u32, u32)>;
```

Comment:
Can you add a comment explaining the fields in the error tuple (I assume matching `(caller_code_len, tombstone_code_len)`). It's not immediately clear. Or could make a `struct` instead of using a tuple.

Potentially applied to other methods here too.